<?xml version="1.0"?>

<launch>
  <arg name="kalman_base" default="false"/>
  <arg name="camera_window" default="false"/>

  <rosparam command="load" file="$(find marsha_cell_safety)/config/thor_safety_mediapipe.yaml" />
  <include file="$(find skeleton_trajectory)/launch/skeleton_trajectory.launch">
      <arg name="kalman_base" value="$(arg kalman_base)"/>
      <arg name="camera_window" value="$(arg camera_window)"/>
  </include>


  <node pkg="pose_transform" type="pose_transform" name="pose_transform"> </node>

  <include  file="$(find marsha_cell_safety)/launch/thor_safety.launch"/>
</launch>
